code_connect.py

```diff
@@ -83,21 +84,12 @@ def main(shell: str = None, max_idle_time: int = MAX_IDLE_TIME):
     # Find the first socket that is open, most recently accessed first
     ipc_sock = next_open_socket(socks)
 
-    # Output a shell string to stdout
-    if shell == 'fish':
-        source_lines = [
-            f'# fish usage: ./code_connect.py | source',
-            f'export VSCODE_IPC_HOOK_CLI="{ipc_sock}"',
-            f'alias code="{code_binary}"'
-        ]
-    else:
-        source_lines = [
-            f'# bash usage: ./code_connect.py | eval',
-            f'export VSCODE_IPC_HOOK_CLI="{ipc_sock}"',
-            f'alias code="{code_binary}"'
-        ]
-
-    print('\n'.join(source_lines))
+    args = sys.argv.copy()
+    args[0] = code_binary
+
+    os.environ["VSCODE_IPC_HOOK_CLI"] = str(ipc_sock)
+    proc = sp.run(args)
```

Comment:
Really clean, I like it.
Does this forward the actual output of the `code_binary` to stdin/stderr, i.e. does it _not_ capture it some way?

Reply:
Yes, stdout/stderr of this child process is connected to the `code_connect.py` process. Example:
```
╰─>$ ./code_connect.py
code 1.53.0

Usage: code [options][paths...]

To read from stdin, append '-' (e.g. 'ps aux | grep code | code -')

...
```